lib/ui.dart

```diff
@@ -68,4 +55,16 @@ class AppComponent {
 
     driver(client, users, process);
   }
+
+  addUser(userName) {
+    users = []..addAll(users)..add(userName);
+    if( users.length > 1 ) {
+      fetchMovies(users);
```

Comment:
At some point we will need to make that additive instead of cumulative so that we don't fetch the same data multiple times unnecessarily.